imap_processing/hit/l2/hit_l2.py

```diff
@@ -446,15 +503,14 @@ def process_summed_intensity_data(l1b_summed_rates_dataset: xr.Dataset) -> xr.Da
     # Add systematic uncertainties to the dataset. These will not
     # have the intensity calculation applied to them
     for var in l2_summed_intensity_dataset.data_vars:
-        if "_" not in var:
-            particle = str(var)
+        if var in VALID_SPECIES:
             l2_summed_intensity_dataset = add_systematic_uncertainties(
                 l2_summed_intensity_dataset,
-                particle,
+                str(var),
```

Comment:
Your VALID_SPECIES is only strings, so you are guaranteed a string here already.
```suggestion
                var,
```